lib/view/templates.py

```diff
@@ -1638,9 +1638,9 @@ def numeric_sum_aggregator_selector(key, is_numeric):
 
 
 def latency_weighted_avg(edatas: list[EntryData]):
-    ops_sec_values: map[float] = map(lambda edata: edata.record["ops/sec"], edatas)
-    pcts = map(lambda data: data[0].value * data[1], zip(edatas, ops_sec_values))
```

Comment:
This is the issue. This pre-processing step should have been removed when the `weighted_avg` func was changed. It is applying the weights beforehand.